input/pagecontent/terminology.md

```diff
@@ -3,6 +3,11 @@ This page lists all ValueSets and CodeSystems defined as part of this specificat
 
 As well as these included definitions this specification typically references [National Clinical Terminology Service (NCTS)](https://www.healthterminologies.gov.au/integration/R4/fhir) terminology resources. 
 
+
+<div class="stu-note">
+Expansions of ValueSets in this Implementation Guide may differ from expansions returned from interactions with a terminology server using the latest version of terminology.hl7.org (THO).
```

Comment:
Tweak suggested to the wording. "Implementers are advised to take note that expansions of value sets visible in this guide may differ from expansions returned with a server using [HL7 Terminology (THO)](http://terminology.hl7.org/6.0.2) version 6.0.0 or higher."

Reply:
Updated wording as per suggestion.